### PR TITLE
test(#313): AT for #255 - prompted for additional information when my booking request has certain attributes

### LINKED ISSUE #313: Acceptance for #255 prompted for additional information when my booking request has certain attributes
### User story
#255

### Acceptance Test Checklist

- [x] User that books a lounge room has its respective attributes prompted
- [x] User that books a mezzanine room has its respective attributes prompted
- [x] User that books a conference room has its respective attributes prompted

### PR DESCRIPTION
**Pull request by a-drew**
##### _What existing problem does the pull request solve?_

Covers #255 as described in AT criterias from #313 

##### _Demonstrate the code is solid. </br> Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI._

`php artisan dusk`

##### _List all issues this PR resolves_
- closes #313

##### _Complete these items before requesting reviews_
- [X] Static Analysis Rules Validated
- [X] All commits follow naming conventions

### Sub-tasks Checklist
- [ ] UI Implementation
- [ ] UI Test
- [ ] Backend Implementation
- [ ] Backend Tests
- [ ] Integration Tests
- [X] System Tests
- [ ] Documentation

### CODE REVIEW

**Review by KeeganWoods1 — APPROVED** [review]
Lgtm!

**Review by EvanDime — APPROVED** [review]

**Comment by EvanDime** [comment]
@a-drew once the tests complete. Can you confirm this is ready to merge? if so please make the merge

File: database/seeders/RolesAndPermissionsSeeder.php [hunk 1]
```diff
@@ -48,11 +48,18 @@ public function run()
...
         foreach ($permissions as $permission) {
-            Permission::create(['name' => $permission]);
+            Permission::updateOrCreate(['name' => $permission]);
         }
 
-        // Create roles
-        Role::create(['name' => 'super-admin'])->givePermissionTo($permissions);
+        foreach ($roles as $role => $givenPermissions) {
+            Role::updateOrCreate(['name' => $role])->syncPermissions($givenPermissions);
+        }
```
Comment (alexstojda) [inline database/seeders/RolesAndPermissionsSeeder.php:63]: Good call, much cleaner and easier to modify

**Review by alexstojda — APPROVED** [review]
Nice job on this one

**Pull request merged**